components/lvgl_esp32_drivers/lvgl_tft/ili9488.c

```diff
@@ -128,11 +128,11 @@ void ili9488_flush(lv_disp_drv_t * drv, const lv_area_t * area, lv_color_t * col
 
     for (uint32_t i = 0; i < size; i++) {
         LD = buffer_16bit[i].full;
-        mybuf[j] = (uint8_t) ((LD & 0xF800) >> 8);
+        mybuf[j] = (uint8_t) (((LD & 0xF800) >> 8) | ((LD & 0x8000) >> 13));
```

Comment:
It might be a little bit more efficien:
```c
 mybuf[j] = (uint8_t) (((LD & 0xF800) >> 8) | ((LD & 0x8000) ? 1 : 0));
```

Reply:
I not sure (not disassembled gcc optimized output for esp32) if "test/and" followed by "conditional **jump**" and "or with constant" is better then "test/and", followed by "rotate" and "or with register".
By the way constant should be "4" not "1".

![ili9488_serial_rgb666](https://user-images.githubusercontent.com/1438115/89872955-fd1a1880-dbb9-11ea-85e0-4511b4a12ca8.jpg)

Reply:
You are right, probably it really compiles to conditional jump so the difference is not so obvious. 

Anyway, the difference should be negligible, so let's leave your solution.